fenicsadapter/config.py

```diff
@@ -46,6 +46,7 @@ def read_json(self, adapter_config_filename):
         except:
             self._write_data_name = None
         self._read_data_name = data["interface"]["read_data_name"]
+        self._data_writing_type = data["interface"]["data_writing_type"]
```

Comment:
I think it is good that we make the `data_writing_type` configurable. What does it default to? Please make sure that it is an optional argument in our configuration file.

Reply:
I think the default should be `"absolute"`, so by default the adapter will write absolute values of the write data. Is that okay?

Reply:
Yes. Totally agree. Please make sure to also document the meaning of these two keywords somewhere.